scripts/lochness_create_template.py

```diff
@@ -157,10 +157,10 @@ def create_keyring_template(keyring_loc: Path, args: object) -> None:
                 'USERNAME': f'**id_for_xnat_{study}**',
                 'PASSWORD': f'**password_for_xnat_{study}**'}
 
-    if 'box' in args.sources:
-        if 'SECRETS' not in template_dict['lochness'].keys():
-            template_dict['lochness']['SECRETS'] = {}
+    if 'SECRETS' not in template_dict['lochness'].keys():
+        template_dict['lochness']['SECRETS'] = {}
 
+    if 'box' in args.sources:
         for study in args.studies:
             template_dict['lochness']['SECRETS'][study] = 'LOCHNESS_SECRETS'
```

Comment:
1. Is `'LOCHNESS_SECRETS'` a global passphrase? It looks like I missed it in earlier PRs.

2. Here are you assigning that global passphrase to all box studies? What if a passphrase already exists in `lochness.SECRETS`:

<details><summary>expand</summary>

```json
{
    "lochness": {

        "REDCAP": {
            "FAKE": {
                "redcap.FAKE": [
                    "FAKE"
                ]
            },
        },

        "SECRETS": {
            "BWH":"existing passphrase",
            "FAKE":""
        }
    },
    
    ...
    ...
}
```

</details>

Reply:
1. `LOCHNESS_SECRETS` is only used in `lochness.box` module, where it encrypts already pulled data with `crypt` module if the option to do so is provided by the user. Not planned to use this function in U24 though.

2. This is a template creation function. `if a passphrase already exists` may mean we already have `lochness.json`. In this situation we will not use the template creation function to create the keyring example.